sceptre/context.py

```diff
@@ -0,0 +1,69 @@
+from os import path
+
+
+class SceptreContext(object):
+
+    def __init__(self, project_path=None, command_path=None,
+                 user_variables=None, options=None, output_format=None,
+                 no_color=None):
+        # project_path: absolute path to the base sceptre project folder
+        # e.g. absolute_path/to/sceptre_directory
+        self.project_path = self._validate_path_has_value(project_path)
```

Comment:
Unless validation is going to happen, this function doesn't really provide one. We might as well omit `None` so that the arg has to be passed. This applies to both `project_path` and `command_path`.